### PR TITLE
Creates a use case scenario documentation

### PR DESCRIPTION
**Pull request by jeffhsta**
@jeffhsta @carolinaborim pairing
Documentation guiding a user journey, mentioning code examples from
Dealership Demo

### CODE REVIEW

File: source/includes/external_user_workflow/_workflow.md [hunk 1]
```diff
@@ -132,3 +133,46 @@ of the following examples available on the right side.
 <aside class="notice">
 All requests should contain an Authorization header.
 </aside>
+
+## Use case scenarios
+
+### Credentials
+
+To do the requests is necessary to have a credential with access to a list of equipment.
```
Comment (andressa) [inline source/includes/external_user_workflow/_workflow.md]: s/requests is necessary/it is necessary/

File: source/includes/external_user_workflow/_workflow.md [hunk 2]
```diff
@@ -132,3 +133,46 @@ of the following examples available on the right side.
...
+There is a [documentation](https://agco-fuse.github.io/documentation/#authenticate-using-oauth2) that explain how the authentication works.
```
Comment (andressa) [inline source/includes/external_user_workflow/_workflow.md]: s/that explain/that explains/

File: source/includes/external_user_workflow/_workflow.md [hunk 3]
```diff
@@ -132,3 +133,46 @@ of the following examples available on the right side.
...
+### Equipment API
+
+### Get a list of equipment
+Given I'm a developer with a Bearer token,<br>
+When I do a request to Equipment API, asking for a list of equipment<br>
+The API will answer with the requested list<br>
+
+**How it works?**
+
+At this [code example](https://github.com/agco-fuse/dealership-demo/blob/master/app/scripts/services/apiservice.js#L11) we are telling the API that we would be able to get a list of equipment.
```
Comment (andressa) [inline source/includes/external_user_workflow/_workflow.md]: s/we are telling the API that we would be able to get a list of equipment/we are requesting the API to get a list of equipment/

File: source/includes/external_user_workflow/_workflow.md [hunk 4]
```diff
@@ -132,3 +133,46 @@ of the following examples available on the right side.
...
+More details can be found on link [Get equipment section](#get-equipment)
+
+### Get a single equipment
+Given I'm a developer with a Bearer token,<br>
+When I do a request to Equipment API, asking for a single equipment<br>
+The API will answer with the requested equipment<br>
+
+**How it works?**
+
+At this [code example](https://github.com/agco-fuse/dealership-demo/blob/master/app/scripts/services/apiservice.js#L21) we are telling the API that we would be able to get a specific equipment when we pass a valid id.
```
Comment (andressa) [inline source/includes/external_user_workflow/_workflow.md]: s/telling the API that we would be able to get a specific equipment when we pass a valid id/requesting the API to get a specific equipment by passing a valid id as a parameter

File: source/includes/external_user_workflow/_workflow.md [hunk 5]
```diff
@@ -7,6 +7,7 @@ to follow these steps:
 2. [Check your e-mail inbox](#check-your-e-mail-inbox)
 3. [Reset your password](#reset-your-password)
 4. [Authenticate](#authenticate)
+5. [Use case scenarios](#use-case-scenarios)
```
Comment (glauco) [inline source/includes/external_user_workflow/_workflow.md]: What do you think about creating a separated file for this particular piece?

File: source/includes/external_user_workflow/_workflow.md [hunk 6]
```diff
@@ -132,3 +133,47 @@ of the following examples available on the right side.
 <aside class="notice">
 All requests should contain an Authorization header.
 </aside>
+
+## Use case scenarios
```
Comment (glauco) [inline source/includes/external_user_workflow/_workflow.md]: s/Use case scenarios/An example: retrieving equipment data/ or something that actually reflects the business use case

File: source/includes/external_user_workflow/_workflow.md [hunk 7]
```diff
@@ -132,3 +133,47 @@ of the following examples available on the right side.
...
+
+### Request an access token
+
+The best way of authentication having credentials in a Bearer token format, there is a [code example](https://github.com/agco-fuse/dealership-demo/blob/master/index.js#L17) showing how to do that.
+
+### Equipment API
+
+### Get a list of equipment
+Given I'm a developer with a Bearer token,<br>
+
```
Comment (glauco) [inline source/includes/external_user_workflow/_workflow.md]: nitpick: remove empty line

File: source/includes/external_user_workflow/_workflow.md [hunk 8]
```diff
@@ -132,3 +133,47 @@ of the following examples available on the right side.
 <aside class="notice">
 All requests should contain an Authorization header.
 </aside>
+
+## Use case scenarios
+
```
Comment (glauco) [inline source/includes/external_user_workflow/_workflow.md]: I believe we could introduce the user to the business use case, then go to the details. What are your thoughts on it?

File: source/includes/external_user_workflow/_workflow.md [hunk 9]
```diff
@@ -132,3 +133,47 @@ of the following examples available on the right side.
...
+### Credentials
+
+To do the requests it is necessary to have a credential with access to a list of equipment.
+There is a [documentation](https://agco-fuse.github.io/documentation/#authenticate-using-oauth2) that explains how the authentication works.
+
+### Request an access token
+
+The best way of authentication having credentials in a Bearer token format, there is a [code example](https://github.com/agco-fuse/dealership-demo/blob/master/index.js#L17) showing how to do that.
```
Comment (lucaslago) [inline source/includes/external_user_workflow/_workflow.md]: s/The best way of authentication/In order to consume our API, a valid bearer token must be set in the Authorization header of every HTTP request/

File: source/includes/external_user_workflow/_workflow.md (hunk 5 above)
Comment (jeffhsta) [inline source/includes/external_user_workflow/_workflow.md]: @glauco Yes, we can do it. Do you think that it should be a separated item or sub item inside 'Getting Started'?

**Comment by waldemarnt** [comment]
👍

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
+# Use case scenarios
+
+## Retrieving equipment data
+
+The AGCO's Telemetry API is a solution where equipment data are centralized and one of it's functionalities is the retrieval of those data.
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/it's/its

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+There is also a façade of Telemetry API, that respond with more information related to the equipment, making it easier to collect telemetry data, such as *status*, *location* and *CAN variables*.
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/respond/responds

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+
+### Credentials
+
+To do the requests it is necessary to have a credential with access to a list of equipment.
+There is a [documentation](https://agco-fuse.github.io/documentation/#authenticate-using-oauth2) that explains how the authentication works.
+
+### Request an access token
+
+The best way of authentication having credentials in a Bearer token format, there is a [code example](https://github.com/agco-fuse/dealership-demo/blob/master/index.js#L17-L30) showing how to do that.
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/The best way of authentication having credentials in a bearer token format /In order to consume our API, a valid bearer token must be set in the Authorization header of every HTTP request. There/

**Comment by lucaslago** [comment]
👍

**Pull request merged**